test/axes/numericAxisTests.ts

```diff
@@ -1,468 +1,343 @@
 ///<reference path="../testReference.ts" />
 
-describe("NumericAxis", () => {
-  function boxIsInside(inner: ClientRect, outer: ClientRect, epsilon = 0) {
-    if (inner.left < outer.left - epsilon) { return false; }
-    if (inner.right > outer.right + epsilon) { return false; }
-    if (inner.top < outer.top - epsilon) { return false; }
-    if (inner.bottom > outer.bottom + epsilon) { return false; }
-    return true;
-  }
-
-  function assertBoxInside(inner: ClientRect, outer: ClientRect, epsilon = 0, message = "") {
-    assert.operator(inner.left, ">", outer.left - epsilon, message + " (box inside (left))");
-    assert.operator(inner.right, "<", outer.right + epsilon, message + " (box inside (right))");
-    assert.operator(inner.top, ">", outer.top - epsilon, message + " (box inside (top))");
-    assert.operator(inner.bottom, "<", outer.bottom + epsilon, message + " (box inside (bottom))");
-  }
-
-  it("tickLabelPosition() input validation", () => {
-    let scale = new Plottable.Scales.Linear();
-    let horizontalAxis = new Plottable.Axes.Numeric(scale, "bottom");
-    assert.throws(() => horizontalAxis.tickLabelPosition("top"), "horizontal");
-    assert.throws(() => horizontalAxis.tickLabelPosition("bottom"), "horizontal");
-
-    let verticalAxis = new Plottable.Axes.Numeric(scale, "left");
-    assert.throws(() => verticalAxis.tickLabelPosition("left"), "vertical");
-    assert.throws(() => verticalAxis.tickLabelPosition("right"), "vertical");
-  });
-
-  it("draws tick labels correctly (horizontal)", () => {
-    let SVG_WIDTH = 500;
-    let SVG_HEIGHT = 100;
-    let svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
-    let scale = new Plottable.Scales.Linear();
-    scale.range([0, SVG_WIDTH]);
-    let numericAxis = new Plottable.Axes.Numeric(scale, "bottom");
-    numericAxis.renderTo(svg);
-
-    let tickLabels = (<any> numericAxis)._element.selectAll("." + Plottable.Axis.TICK_LABEL_CLASS);
-    assert.operator(tickLabels[0].length, ">=", 2, "at least two tick labels were drawn");
-    let tickMarks = (<any> numericAxis)._element.selectAll("." + Plottable.Axis.TICK_MARK_CLASS);
-    assert.strictEqual( tickLabels[0].length, tickMarks[0].length, "there is one label per mark");
-
-    let i: number;
-    let markBB: ClientRect;
-    let labelBB: ClientRect;
-    for (i = 0; i < tickLabels[0].length; i++) {
-      markBB = tickMarks[0][i].getBoundingClientRect();
-      let markCenter = (markBB.left + markBB.right) / 2;
-      labelBB = tickLabels[0][i].getBoundingClientRect();
-      let labelCenter = (labelBB.left + labelBB.right) / 2;
-      assert.closeTo(labelCenter, markCenter, 1, "tick label is centered on mark");
+describe("Axes", () => {
+  describe("NumericAxis", () => {
+    function boxIsInside(inner: ClientRect, outer: ClientRect, epsilon = 0) {
```

Comment:
if this is only used in assertion, probably remove this and use `assertBoxInside` instead